homeassistant/components/google_tasks/config_flow.py

```diff
@@ -55,4 +69,30 @@ async def async_oauth_create_entry(self, data: dict[str, Any]) -> FlowResult:
         except Exception as ex:  # pylint: disable=broad-except
             self.logger.exception("Unknown error occurred: %s", ex)
             return self.async_abort(reason="unknown")
-        return self.async_create_entry(title=self.flow_impl.name, data=data)
+        user_id = user_resource_info["id"]
+        if not self.reauth_entry:
+            await self.async_set_unique_id(user_id)
+            self._abort_if_unique_id_configured()
+            return self.async_create_entry(title=self.flow_impl.name, data=data)
```

Comment:
Maybe its worth including the account name in the title so users know which account they are expected to login to.

Reply:
Good one